charts/openshift-metering/values.yaml

```diff
@@ -31,6 +31,14 @@ storage:
   type: "hive"
   hive: {}
 
+
+  config:
+    db:
+      url: ""
+      driver: ""
+      username: ""
+      password: ""
+      secret: ""
```

Comment:
No need to redefine this configuration - just add the `secretName` field to the existing `spec.hive.spec.config.db`.

Reply:
I found hive.spec.config.db at line 692